src/dbConn.php

```diff
@@ -0,0 +1,19 @@
+<?php
+namespace TheRealMVP;
+use PDO;
+
+/**Connects to database
+ *
+ * No params given
+ *
+ * @return PDO
+ */
+class dbConn
```

Comment:
I think this needs to not be a class, but the function needs to be a public function inside the TeamHydrator class?

Reply:
Also, I'm fairly sure that the connection is what Rich and Elmi are working on, so in theory we should be able to use part of their code to connect to DB and then pass that connection in when doing your getData request? So we may not need this function once Rich and Elmi have finished.

Reply:
Hmm, we should probably create another class if we want to share the connection in multiple places...

Reply:
As discussed, leave for now